draft-ietf-rtcweb-jsep.xml

```diff
@@ -1996,13 +1998,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <list style="symbols">
 
               <t>An "a=mid" line, as specified in
-              <xref target="RFC5888"></xref>, Section 4. All mid values
+              <xref target="RFC5888"></xref>, Section 4. All MID values
```

Comment:
Is MID capitalized or not? Not sure we've been consistent on this, although I think I prefer MID to mid for consistency with RID.